src/app/pages/home.js

```diff
@@ -7,11 +8,21 @@ import WishListContainer from '../components/WishList/WishListContainer';
 
 const HomePage = () => (
   <>
```

Comment:
we shouldn't set flag for each component

please, leave only 2 flags here
'wishlist', 'advertising'

as you can see I suggested to use shorter names for the flag

Reply:
completed